assets/css/less/admin.less

```diff
@@ -9,11 +9,14 @@
 	--orange: #F15A24;
 }
 
+@import 'colors.less';
 @import 'common.less';
 @import 'fonts.less';
+@import 'buttons.less';
```

Comment:
I think this would be more clear if we call is admin-buttons since we'll only add this styling on the back-end.